index.js

```diff
@@ -162,6 +169,7 @@ function createHarness(conf_) {
 
     var only = false;
     test.only = function () {
+        if (conf_.noOnly) { throw new Error('there should be no only test'); }
         if (only) { throw new Error('there can only be one only test'); }
```

Comment:
```suggestion
        if (only) { throw new Error('there can only be one `only` test'); }
        if (conf_.noOnly) { throw new Error('`only` tests are disabled'); }
```
let's throw the "bug" one first, and the "disabled" one second

Reply:
done, although I'm thinking `disabled` makes it sound like the `--no-only` flag disables the `.only` tests and would run just fine? 

was thinking maybe `not allowed` or `disallowed`? Do let me know your thoughts on it. Thanks

Reply:
Good point. Maybe “are prohibited’?

Reply:
okay, sounds good, captures the motive. Making the update then, thanks

Reply:
done